sos/report/plugins/redis.py

```diff
@@ -46,26 +60,18 @@ def setup(self):
             ])
 
     def postproc(self):
-        self.do_file_sub(
-            "/etc/redis.conf",
-            r"(masterauth\s).*",
-            r"\1********"
-        )
-        self.do_file_sub(
-            "/etc/redis.conf",
-            r"(requirepass\s).*",
-            r"\1********"
-        )
-        self.do_path_regex_sub(
-            self.var_puppet_gen + "/etc/redis.conf*",
-            r"(masterauth\s).*",
-            r"\1*********"
-        )
-        self.do_path_regex_sub(
-            self.var_puppet_gen + "/etc/redis.conf*",
-            r"(requirepass\s).*",
-            r"\1*********"
-        )
-
-
+        for path in ["/etc/",
+                     self.var_puppet_gen + "/etc/",
+                     "/etc/opt/rh/rh-redis32/",
+                     "/etc/opt/rh/rh-redis5"]:
```

Comment:
Shouldnt be there "/etc/opt/rh/rh-redis5/" instead? If we call `+` to strings, we need the dir separator explicitly written :).